prostir/http/models.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import MetaData, Table, Column, Integer, String, Float, Null, TIMESTAMP
+
+metadate = MetaData()
+
+users = Table(
+    "users",
+    metadate,
+    Column("id", Integer, primary_key=True),
+    Column("number", String),
```

Comment:
Краще переіменувати на `phone`
```suggestion
    Column("phone", String),
```